src/main/java/com/marklogic/client/modulesloader/impl/BaseModulesFinder.java

```diff
@@ -31,6 +31,8 @@
     private String optionsPath = "options";
     private String namespacesPath = "namespaces";
     private String transformsPath = "transforms";
+    // special case for excluding from assets.
+    private String schemasPath = "schemas";
```

Comment:
It took me a bit to figure out what's going on here.  Fortunately, it's a confusion of the testing space, not the implementation.  For this project, I tested the SchemasFinder with the same baseDir as modules, and so the new directory for schemas was getting picked up by the ModulesFinder too.  In actuality (for ml-gradle and for ml-app-deployer), the schemas `baseDir` is `src/main/ml-config/schemas`, that is, alongside database configuration.  I think you're completely right that schemas should not be under ml-modules, and I'll separate and fix the test cases to reflect that.

Timing of 2.0 release is fine for me.  My schemas work is toward Markogic 9 feature, and I just need it internally for the time being.  After you release, I'll make a new PR with whatever version numbers you specify.  That will be toward the end of this month.